src/DIRAC/RequestManagementSystem/private/RequestTask.py

```diff
@@ -290,11 +286,15 @@ def __call__(self):
           for opFile in operation:
             opFile.Status = 'Failed'
           operation.Status = 'Failed'
+      elif userSuspended in setupProxy['Message']:
+        # If user is suspended, wait for a long time
+        self.request.NotBefore = datetime.datetime.utcnow() + datetime.timedelta(hours=6)
```

Comment:
you can use `Request.delayNextExecution`

Reply:
I've made the review commends in the hope of getting most PRs merged before tomorrow's reformatting mess.